tosca/core/src/main/java/nl/jads/tosca/DefectPredictorKBApi.java

```diff
@@ -56,7 +57,11 @@ public DefectPredictorKBApi(KB kb, String homeLoc) {
     public static void main(String[] args) throws IOException {
         DefectPredictorKBApi kbApi = new DefectPredictorKBApi(new KB());
         FindBugInput findBugInput = new FindBugInput();
+        long startTime = Instant.now().toEpochMilli();
```

Comment:
I just added this time library so as to have this in the file, in case we want to count time.